### PR TITLE
Op. Mode column in Lifts

### PR DESCRIPTION
**Pull request by Angatupyry**
## What's new

- Op. Mode column added

Screenshot from 2023-06-13 13-36-14

## Self-checks

- [ ] I have prototyped this new feature (if necessary) on Figma 
- [ ] I'm familiar with and follow this [Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

### CODE REVIEW

File: packages/dashboard/src/components/lifts-app.tsx [hunk 1]
```diff
@@ -25,33 +26,40 @@ export const LiftsApp = createMicroApp('Lifts', () => {
...
+        const { health_status } = data;
+        const sub = rmf.getLiftStateObs(lift.name).subscribe((liftState) => {
+          setLiftTableData((prev) => {
+            return {
+              ...prev,
+              [lift.name]: [
+                {
+                  index: i,
+                  name: lift.name,
+                  opMode: health_status ? health_status : 'N/A',
```
Comment (aaronchongth) [inline packages/dashboard/src/components/lifts-app.tsx:41]: I think we'll need to remap the text, https://github.com/open-rmf/rmf-web/blob/main/packages/api-server/api_server/models/health.py#L10, from Healthy, Unhealthy, Dead to ONLINE, UNSTABLE, OFFLINE, with a similar styling with the door status (colored font, uppercase)

**Review by aaronchongth — CHANGES_REQUESTED** [review]
Thanks for getting this out so quickly! I left one review comment which will apply to doors too

File: packages/dashboard/src/components/lifts-app.tsx (hunk 1 above)
Comment (aaronchongth) [reply packages/dashboard/src/components/lifts-app.tsx:41]: image

Comment (Angatupyry) [reply packages/dashboard/src/components/lifts-app.tsx:41]: Nice!
Done here 7bf9c9673c38fd08fa38d14f1e5e81b2b178a5b0

**Review by aaronchongth — APPROVED** [review]
thanks! LGTM

**Pull request merged**